GalaxyZooTouchTable/GalaxyZooTouchTable/Models/GalaxyRing.cs

```diff
@@ -0,0 +1,95 @@
+﻿using GalaxyZooTouchTable.ViewModels;
+using System.Windows.Media.Imaging;
+
+namespace GalaxyZooTouchTable.Models
+{
+    public class GalaxyRing : ViewModelBase
+    {
+        const int RING_WIDTH_STEP = 16;
+        const int AVATAR_WIDTH_STEP = 6;
+        const int INITIAL_RING_WIDTH = 56;
+        const int INITIAL_AVATAR_RADIUS_POSITION = 20;
+
+        public BitmapImage UserAvatar { get; set; }
+        public string UserColor { get; set; } = "#E5FF4D";
+        public TableUser User { get; private set; }
+
+        private int _avatarX = 0;
+        public int AvatarX
+        {
+            get => _avatarX;
+            set => SetProperty(ref _avatarX, value);
+        }
+
+        private int _avatarY = 0;
+        public int AvatarY
+        {
+            get => _avatarY;
+            set => SetProperty(ref _avatarY, value);
+        }
+
+        private int _diameter = 0;
+        public int Diameter
+        {
+            get => _diameter;
+            set => SetProperty(ref _diameter, value);
+        }
+
+        private bool _currentlyClassifying = true;
+        public bool CurrentlyClassifying
+        {
+            get => _currentlyClassifying;
+            set => SetProperty(ref _currentlyClassifying, value);
+        }
+
+        public GalaxyRing(int index = 0, TableUser user = null)
+        {
+            if (user != null)
+            {
+                User = user;
+                UserColor = user.ThemeColor;
+                UserAvatar = user.Avatar;
```

Comment:
I think you should either store all of `user`, or store all of the values of `user` that you need.